### PR TITLE
Reduce number of errors from 6 to 2; fix >60 exc warnings

### PR DESCRIPTION
**Pull request by losvald**

### CODE REVIEW

File: src/library/scala/collection/immutable/Set.scala [hunk 1]
```diff
@@ -26,17 +26,17 @@ import parallel.immutable.ParSet
  *  @define coll immutable set
  */
 trait Set[A] extends Iterable[A]
-                with GenSet[A]
+//                with GenSet[A]
```
Comment (TiarkRompf) [inline src/library/scala/collection/immutable/Set.scala:29]: why is this necessary?

Comment (losvald) [inline src/library/scala/collection/immutable/Set.scala:29]: It's not changing any of our errors, and this is how it was in the first
place. So, IMHO it's better to leave it.

**Comment by losvald** [comment]
Fixed the majority of exc warnings. What has remained is mostly related to process.BasicIO (1/3) (hard to fix), control control pkg (1/3) and some collections that use IO & synchronization (1/4).
